app/controllers/articles_controller.rb

```diff
@@ -43,8 +43,13 @@ def new
     @tag = Tag.find_by_name(params[:template])
     @article = if @tag.present? && @user&.editor_version == "v2"
                  authorize Article
-                 Article.new(body_markdown: "", cached_tag_list: @tag.name,
-                             processed_html: "", user_id: current_user&.id)
+                 if @tag&.submission_template.present?
```

Comment:
Hi, @aspittel 
I think this code could be refactored to smth like this, not to increase nesting:
```ruby
body_markdown = @tag.submission_template? ? @tag.submission_template_customized(@user.name) : ""
Article.new(body_markdown: body_markdown, cached_tag_list: @tag.name, processed_html: "", user_id: current_user&.id)
```
I also think there's no need to use `&` for `@tag`, cause there is a check `@tag.present?` before.
I would also remove `&` on `current_user&.id` cause the action is allowed for the authorized users only, so `current_user` exists anyway.